### ISSUE TITLE
Too easy to inadvertantly select message content while scrolling

### ISSUE THREAD

**Issue by paulburton0** [thunderbird/thunderbird-android#6005]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

5.905

### Where did you get the app from?

F-Droid

### Android version

12

### Device model

Samsung Galaxy S20 Ultra 5G (SM-G988U)

### Steps to reproduce

Open any email and scroll the message content (it's obviously easier to repro if it's a lengthy message). If you "drag" your finger across the screen to scroll (i.e. instead of "flicking" it)  and move too slowly, the contextual pop-up window appears much too "easily". It seems to be more likely to happen in emails with images and/or hyperlinks than text alone.

### Expected behavior

The context pop up that appears on "long tap" should have more of a delay and/or should only appear if the finger is tapped (i.e. no scrolling).

### Actual behavior

The pop-up menu seems to appear much more "easily" than with other scroll-intensive apps (e.g. Twitter, text editors, browsers). This makes scrolling and reading longer messages (especially those with lots of images and links) rather annoying, because the menu appears and blocks the content.

**Comment by cketti** [comment]
K-9 Mail doesn't do anything special. The long-press timeout should be the same as it is in other apps.

I can't reproduce this behavior on any of my test devices. We haven't gotten a report like this before either. I don't think this is a bug in K-9 Mail. Chances are this is an issue specific to your device. You could try uninstalling, then reinstalling the app. If that doesn't work, you could try a factory reset.

**Comment by giulioportioli** [comment]
Hi @cketti, I'm surprised this has never been reported before. I've noticed this behaviour since I can remember ( > 6 years) and at times I even thought about switching to another email app, so much is irritating (but no ;)
I finally decided to report it, and ended up here after a little googling.

The issue happens frequently but not 100% of the times, mind you. I believe the key is the slow drag:

1. Open an email that is long and contains an image which is also a hyperlink.
2. Tap on the image and immediately scroll up _slowly_, holding the finger down.

The moment you tap on the image, it becomes selected (shaded in blue), then most of the times the scrolling motion immediately cancels the selection, and all is good.

Many times, however, the scrolling motion unexpectedly does _not_ cancel the selection, and within 500 ms or so the context menu pops up (while still scrolling), which is just a nuisance, since you are just scrolling the email to read it.

Basically, tapping on an image and then dragging it upward (as opposed to flick it up) sometimes results in a brief flash of blue and nothing else (selection correctly cancelled) but many times results in an unwanted context menu popping up on what you are trying to read.

I'm currently using K-9 v. 6.0.0.0, Android 12 on Galaxy s10e.
